index.html

```diff
@@ -7,10 +7,11 @@
     <title>Amber Is Awesome</title>
   </head>
   <body>
+    <h1>This a Circle</h1>
```

Comment:
This is nice but if we were being strict about things it would need to go into a different PR.  I don't think we should worry about it right now though.